include/config/mousebind.h

```diff
@@ -0,0 +1,41 @@
+#ifndef __LABWC_MOUSEBIND_H
+#define __LABWC_MOUSEBIND_H
+
+#include <wayland-util.h>
+
+enum mouse_context {
+	MOUSE_CONTEXT_TITLEBAR,
+	MOUSE_CONTEXT_NONE
+};
+
+enum mouse_button {
```

Comment:
These are defined in ‘#include <linux/input-event-codes.h>’
E.g. BTN_RIGHT, etc
So, would a uint32_t be enough?
Doing it this way we’d be consistent with wlroots’s ‘ struct wlr_event_pointer_button’ event->button

Reply:
done